src/main/scala/com/yotpo/metorikku/output/writers/file/CSVOutputWriter.scala

```diff
@@ -5,7 +5,7 @@ import com.yotpo.metorikku.configuration.job.output.File
 import com.yotpo.metorikku.output.Writer
 import org.apache.spark.sql.DataFrame
 
-class CSVOutputWriter(var props: Map[String, String], outputFile: Option[File]) extends Writer {
+class CSVOutputWriter(var props: Map[String, Any], outputFile: Option[File]) extends Writer {
```

Comment:
why is it for?

Reply:
It's a leftover from debugging saving things to S3. I left it because it is more correct this way (the final map is `[String, Any]` and not `[String, String]`